api/v1/class.PasswordResetEmail.php

```diff
@@ -18,7 +18,7 @@ public function getHTMLBody()
     {
         return 'Someone (quite possibly you) has requested a password reset of your Flipside account.<br/>
                 To reset your password click on the link below.<br/>
-                <a href="https://profiles.burningflipside.com/change.php?hash='.$this->user->getPasswordResetHash().'">Reset Password</a><br/>
+                <a href="'.getenv('PROFILES_URL').'/change.php?hash='.$this->user->getPasswordResetHash().'">Reset Password</a><br/>
```

Comment:
Why not just use Settings like everything else? Is there a good reason to have two different mechanisms?